etc/kayobe/ansible/pulp-image-upload.yml

```diff
@@ -5,17 +5,18 @@
     remote_pulp_url: "{{ stackhpc_release_pulp_url }}"
     remote_pulp_username: "{{ stackhpc_image_repository_username }}"
     remote_pulp_password: "{{ stackhpc_image_repository_password }}"
-    repository_name: "kayobe-images-{{ openstack_release }}-{{ os_distribution }}-{{ os_release }}"
-    pulp_base_path: "kayobe-images/{{ openstack_release }}/{{ os_distribution }}/{{ os_release }}"
+    repository_name: "{{ image_name }}-{{ openstack_release }}-{{ os_distribution }}-{{ os_release }}"
+    pulp_base_path: "{{ image_name }}/{{ openstack_release }}/{{ os_distribution }}/{{ os_release }}"
```

Comment:
I think the term `image_name` is a bit misleading. Something like `image_path` or `image_directory` might make more sense

Reply:
Superseded by other comments